student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,12 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+     const char* username = NULL;
+     username = my_username();
+     
+     char* malloc_username = NULL;
+     malloc_username = malloc_username_from_conf_file();
+     
+
+    TEST_ASSERT_TRUE_MESSAGE(username,malloc_username);
```

Comment:
@samu7988 message is missing here. also you should free the string referenced by `malloc` here.